readme.md

```diff
@@ -11,12 +11,13 @@ It does only one thing and does it very well - provides a possibility to execute
 
 It *does not* break http semantics.
 
-See live demo [*HERE*](https://runkit.com/jasonrammoray/5a8141d8ac0ed40013fd9054).
+See live demo [*HERE*](https://runkit.com/l2jliga/5b2c00bf17259600121348f2).
```

Comment:
@L2jLiga, why would we want to force our users towards specific style usage?
The thing is that when taking something directly from rxjs outside the scope of this library means, that you automatically taking a responsibility to always know which version of rxjs is being used either in httpRx (in case if project doesn't depend on rxjs directly), or in both httpRx and your project. 
If a programmer understands that and want to use it that way, it is ok.
On the other hand one might want to stick with chainable methods (e.g. `map`, `catch`, `finally`, etc).
It is up to developer to decide on a style choice.

We can place both examples in the README one after another.

Reply:
@whatdoyouseedoc , @blackorion , any thoughts from you guys?

Reply:
@JasonRammoray, in RxJS 5 and 6 we can use `pipe` operator but we can't use chains in RxJS 6.
If user don't installed specific version of RxJS then he will use RxJS 6 and if he used chain instead of `pipe` operator then he will get error.

Reply:
How about to mark jasonrammoray's demo like "Doesn't work in RxJS6", l2jliga's demo as preferable and left both demos?

Reply:
@L2jLiga , got it.
Then it make sense to keep up with a latest changes and to raise a major version to indicate backwards incompatibility.

Reply:
@L2jLiga , please do npm version major and commit your changes.

Reply:
@JasonRammoray done